lib/jekyll/liquid_renderer.rb

```diff
@@ -7,8 +7,8 @@ module Jekyll
   class LiquidRenderer
     extend Forwardable
 
-    def_delegator :@site, :in_source_dir, :source_dir
-    def_delegator :@site, :in_theme_dir, :theme_dir
+    private def_delegator :@site, :in_source_dir, :source_dir
+    private def_delegator :@site, :in_theme_dir, :theme_dir
```

Comment:
I recognize that these were always _meant_ to be private, but at this point would it be easier to punt this until `v4.0.0`? Or is this necessary for this PR?

Reply:
These attributes have not been released yet.. I *added them in #6610*..
So they can be safely altered..

Reply:
(this is therefore just a `fix` as wel..)